dozer-cli/src/cli/types.rs

```diff
@@ -22,6 +22,8 @@ pub struct Cli {
     pub config_path: String,
     #[arg(global = true, long)]
     pub config_token: Option<String>,
+    #[arg(global = true, short = 'e', long, default_value = "0")]
+    pub err_threshold: String,
```

Comment:
Can be a `u32` or something?

Reply:
just in case of people writing wrong values, it is better to parse it as String and setting default as 0 if it's invalid

Reply:
I think it's better if people get noticed about their typo instead of we silently ignore that error and default to 0, and leave people wondering why their `21O` threshold didn't work.

Reply:
what do you mean by 210? we set the max err threshold as well then?

Reply:
Haha, that's the letter O followed by 21, used to explain the point.

Reply:
Ah got it haha